unittest/unittest.py

```diff
@@ -180,6 +180,12 @@ def __init__(self):
     def wasSuccessful(self):
         return self.errorsNum == 0 and self.failuresNum == 0
 
+
+def print_exc():
```

Comment:
Personal opinion: no need to obfuscate, line is short enough, just `print_exception` is ok?

Reply:
applied all comments